roles/installer/templates/configmaps/config.yaml.j2

```diff
@@ -42,6 +42,15 @@ data:
     SYSTEM_TASK_ABS_CPU = '{{ task_resource_requirements["limits"]["cpu"] }}'
 {% endif %}
 
+    # Set callback receiver workers based off cpu limit, default workers are 4, but if we have more than 4 cpu we can set higher value for workers
+    {% set cpu_limit = task_resource_requirements["limits"]["cpu"] if "limits" in task_resource_requirements and "cpu" in task_resource_requirements["limits"] -%}
+    {%- if cpu_limit is defined -%}
+        {% set callback_receiver_cpu = cpu_limit | cpu_string_to_decimal %}
+            {%- if callback_receiver_cpu |int > 4 -%}
+                JOB_EVENT_WORKERS = {{ callback_receiver_cpu }}
```

Comment:
Won't this line be incorrectly indented in the rendered file?

Reply:
My bad. Corrected. Thanks @shanemcd